x-pack/winlogbeat/module/powershell/config/winlogbeat-powershell.js

```diff
@@ -84,8 +84,17 @@ var powershell = (function () {
         });
     };
 
+    // countChunksDelimitedBy will return the number of chunks contained in a field
+    // that are delimited by the given delimiter.
+    var countChunksDelimitedBy = function(evt, fromField, delimiter) {
+        return evt.Get(fromField).split(delimiter).length-1;
```

Comment:
I'm probably being paranoid.  But if evt.Get returns null, would the split cause an exception?

Reply:
This is a good point. I did not do any checks because it is only called with fields we know exist, but this might not be the case in the future, will change it to at least not scream in case of null 👍